rfcs/text/0001-rfc-process.md

```diff
@@ -70,26 +88,66 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
+
+   1. Fork the HumanCellAtlas `dcp-community` repository
+   2. Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`. Make `my-feature` descriptive but do not assign an RFC number yet.
+   3. Fill out the Problem Statement and Author(s) sections.
+   4. Push the changes to your branch and create a _Draft_ PR.
+   5. Tag the PR with the "rfc-draft" label.
+
+#### **Non-Authors**:
+
+When an RFC _Draft_ PR is created and furthermore labeled with "rfc-draft" commentary is not allowed since the RFC is under active development. This time period is specifically an opportunity for collaboration on authorship so if an RFC sparks interest and you would like to collaborate on it (especially if you have been thinking about the problem statement yourself), reach out the Author(s) directly and offline. Specific commentary whether agreement or disagreement on the RFC is discouraged until the Author(s) is/are ready for the PR to be proposed/reviewed.
+
+**ADDITION ENDS**
+
 ### Proposing RFCs
 
 #### **Authors**:
 
-*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+**UPDATE BEGINS**
 
-  - Fork the HumanCellAtlas dcp-community repository
-  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
-  *- where "my-feature" is descriptive. Don't assign an RFC number yet.*
+~~*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*~~
+
+Once the RFC has been filled out to the Author's satisfaction, an RFC may be _proposed_ for comment.
+
+
+
+  ~~- Fork the HumanCellAtlas dcp-community repository~~
   
-  - Fill in the RFC with attention to detail
+  ~~- Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md` where "my-feature" is descriptive. Don't assign an RFC number yet.~~
 
+**UPDATE ENDS**
+  
+  - Fill in the RFC with attention to detail
+  
   - Submit a pull request.
   
       For software RFCs:
       - Add the "rfc-community-review" and the appropriate software project name _(such as "Data Store")_ labels. When the RFC impacts multiple DCP software projects, then the "Architecture" label **MUST** be the project name. *If uncertain about the appropriate project name, then **_Ask a PM_** on the HCA **#dcp-project-mgmt** slack channel*
 
       For governance RFCs:
         
-      - Add the "rfc-community-review" and the "Governance" labels. 
+      - Add the "rfc-community-review" and the "Governance" labels.
+      
+      **ADDITION BEGINS**
+      
+      - If the "rfc-draft" label was attached to the PR, remove it.
+      
+      - Find the created ticket in the Zenhub board under `dcp-community` and assign all Author(s) to the ticket.
+      
+      - If the RFC is "non-informational," link the PR ticket to the open ticket that the RFC addresses.
```

Comment:
What does _link the PR ticket to the open ticket_ mean? For example - https://github.com/HumanCellAtlas/dcp-community/issues/107

Reply:
I tried to make this more clear but feel free to re-open if you have a better wording.. this one was tricky for me.